Bunny_Ears.rb

```diff
@@ -0,0 +1,21 @@
+=begin 
+We have a number of bunnies and each bunny has two big floppy ears. 
+We want to compute the total number of ears across all the bunnies recursively (without loops or multiplication).
+bunnyEars(0) → 0
+bunnyEars(1) → 2
+bunnyEars(2) → 4
+=end
+def bunnyEars(bunny)
+    return 0 if bunny == 0   
+    return 2 if bunny == 1
+    ears = 2
+    return ears + bunnyEars(bunny-1)

```

Comment:
```suggestion
   2 + bunnyEars(bunny - 1)
```

Reply:
this one is very good. just one small change.

Reply:
we can technically save space by not storing a variable. so avoid storing things into variables if we are only ever going to use it once and it does not help with understanding it better, then we can just use the value instead of saving it in a variable.